README.md

```diff
@@ -65,9 +65,21 @@ walletWindow.postMessage({
     network: 'https://api.beta.testnet.solana.com',
   },
 }, WALLET_URL);
+
+or 
+
+walletWindow.postMessage({
+  method: 'confirmTX',
+  params: {
+    description: "Description of tx",
+    format: 'JSON',
+    transaction: "Your tx in JSON",
```

Comment:
Would be nice to describe this more.  Is this a JSON array of octets representing the signed wire transaction, or some other kind of JSON representation

Reply:
done